client/src/App.js

```diff
@@ -3,9 +3,9 @@
 import "./App.css";
 import "bootstrap/dist/css/bootstrap.css";
 
+import ManageCustomer, { CustomerData } from "./components/ManageCustomer";
```

Comment:
I don't see `CustomerData` used in this file, should it be removed?

Reply:
Oh, yes. It was a part of my old try. I will remove this. Sorry.